### DISCUSSION TITLE
Add a Custom Health-check URL Option in Observatory

### DISCUSSION THREAD

**Discussion by knano** [v2fly/v2ray-core#1085] [Ideas]
Currently the observatory only support using the predefined `api.v2fly.org` to perform health check. Is it possible to add an option in the configuration file to allow customizing this health-check URL?
Thanks.

**Comment by xiaokangwang** [comment]
Yes, this will be supported. Modification of health check URL will be supported in a later version.

**Comment by 4-FLOSS-Free-Libre-Open-Source-Software** [comment]
Yes, in V2Ray 5.1.0:

```
"observatory": {
    // destination url, need 204 for success return
    "probeURL": "https://connectivitycheck.gstatic.com/generate_204"
}
```
